Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource_printer_test.go

```diff
@@ -971,10 +1074,22 @@ func TestPrintHumanReadableWithNamespace(t *testing.T) {
 		},
 	}
 
+	printOptions := PrintOptions{
+		NoHeaders:          false,
+		WithNamespace:      true,
+		WithKind:           false,
+		KindName:           "",
+		Wide:               false,
+		ShowAll:            false,
+		ShowLabels:         false,
+		AbsoluteTimestamps: false,
+		ColumnLabels:       []string{},
+	}
+
```

Comment:
We can get rid of this and just use `nil` in all spots below to get a printer with default options.